go-controller/pkg/ovn/master.go

```diff
@@ -227,33 +124,11 @@ func (oc *Controller) upgradeOVNTopology(existingNodes *kapi.NodeList) error {
 // TODO: Verify that the cluster was not already called with a different global subnet
 //
 //	If true, then either quit or perform a complete reconfiguration of the cluster (recreate switches/routers with new subnet values)
-func (oc *Controller) StartClusterMaster() error {
+func (oc *DefaultNetworkController) StartClusterMaster() error {
 	klog.Infof("Starting cluster master")
```

Comment:
I would move `klog.Infof("Starting cluster master")` to NetworkControllerManager.Start

I would rename `StartClusterMaster` to `Init` and move the method to the default network controller file.

Reply:
I just deleted `klog.Infof("Starting cluster master")` as it is unclear to me what does it mean. If I move to `NetworkControllerManager.Start` it changed the current behavior that only leader logs the message. If you feel strongly to keep this message, can we add it in a later PR if otherwise the code is ok?

Reply:
What I meant is that if you placed this `klog.Infof("Starting cluster master")` just after `klog.Infof("Won leader election; in active mode")` in `NetworkControllerManager.Start` it would be logged exactly in the same condition where it was logged before.

Reply:
done. thanks!